addons/binding/org.openhab.binding.meteostick/ESH-INF/thing/thing-types.xml

```diff
@@ -21,7 +21,10 @@
 				<options>
 					<option value="0">North America (915MHz)</option>
 					<option value="1">Europe (868MHz)</option>
-					<option value="2">Australia (915MHz)</option>
+					<option value="2">Australia</option>
+					<option value="3">Fine Offset (915MHz)</option>
```

Comment:
If the labels are exactly the same as what is found in the manufacturer documentation, it should be fine. But can you confirm that it indeed occurs there with and without band information?